NEWS.rst

```diff
@@ -2,6 +2,21 @@
 Release Notes
 =============
 
+----------------------------
+pkgcore 0.12.13 (2022-08-09)
+----------------------------
+
+- prototype.tree: fix bug with checking path is contained under a repository,
```

Comment:
Fix checking whether a path is contained within a known repository (specifically: fix the test to not check if path is prefixed with a repository's location. Instead, we check the full path). Previously, `/usr/portage-dir` would wrongly  be considered within `/usr/portage`.